### ISSUE TITLE
MarkdownParser problem

### ISSUE THREAD

**Issue by AndyWu2015** [gamerson/liferay-blade-cli#33]
the BREAKING_CHANGES.markdown file was updated and they added "[](id=xxx)" in href for jumping to some block , but I found that our parser can't parse this feature correctly. It just generates wrong href content , can we fix that ?

**Comment by gamerson** [comment]
Yes please try to fix it, we need the MarkdownParser to work correctly.
You can send them a pull request.

On Wed, May 25, 2016 at 3:52 AM, Andy Wu <notifications@github.com> wrote:

> the BREAKING_CHANGES.markdown file was updated and they added "
> <http://id=xxx>" in href for jumping to some block , but I found that our
> parser can't parse this feature correctly. It just generates wrong href
> content , can we fix that ?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/gamerson/liferay-blade-cli/issues/33>
>



-- 
Greg Amerson
Liferay Developer Tools
Liferay, Inc. www.liferay.com

**Comment by AndyWu2015** [comment]
got it

**Comment by AndyWu2015** [comment]
but I can't find the source code of com.liferay.markdown.converter in liferay-portal repo ,where should I sent the pull request to ?

**Comment by gamerson** [comment]
Its in the liferay-docs repo

On Thu, May 26, 2016 at 12:49 AM, Andy Wu <notifications@github.com> wrote:

> but I can't find the source code of com.liferay.markdown.converter in
> liferay-portal repo ,where should I sent the pull request to ?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/gamerson/liferay-blade-cli/issues/33#issuecomment-221783532>
>



-- 
Greg Amerson
Liferay Developer Tools
Liferay, Inc. www.liferay.com

**Comment by gamerson** [comment]
https://github.com/liferay/liferay-docs/

On Thu, May 26, 2016 at 12:53 AM, Greg Amerson <gregory.amerson@liferay.com>
wrote:

> Its in the liferay-docs repo
>
> On Thu, May 26, 2016 at 12:49 AM, Andy Wu <notifications@github.com>
> wrote:
>
>> but I can't find the source code of com.liferay.markdown.converter in
>> liferay-portal repo ,where should I sent the pull request to ?
>>
>> —
>> You are receiving this because you commented.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/gamerson/liferay-blade-cli/issues/33#issuecomment-221783532>
>>
>
>
>
> --
> Greg Amerson
> Liferay Developer Tools
> Liferay, Inc. www.liferay.com
>



-- 
Greg Amerson
Liferay Developer Tools
Liferay, Inc. www.liferay.com

**Comment by AndyWu2015** [comment]
Are you sure ? I didn't find it.

**Comment by gamerson** [comment]
/dev/repos/liferay/liferay-docs(master ✔) find . | grep Markdown
./code/liferay-doc-utils/src/com/liferay/documentation/util/AddMarkdownChTemplate.java
./code/liferay-doc-utils/src/com/liferay/documentation/util/AddMarkdownMetadata.java
./code/liferay-doc-utils/src/com/liferay/documentation/util/ConcatMarkdown.java
./code/liferay-doc-utils/src/com/liferay/documentation/util/MarkdownToHtmlDebuggerMain.java
./code/liferay-doc-utils/src/com/liferay/documentation/util/MarkdownToHtmlMain.java

On Thu, May 26, 2016 at 12:58 AM, Andy Wu <notifications@github.com> wrote:

> Are you sure ? I didn't find it.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/gamerson/liferay-blade-cli/issues/33#issuecomment-221784537>
>



-- 
Greg Amerson
Liferay Developer Tools
Liferay, Inc. www.liferay.com

**Comment by AndyWu2015** [comment]
Yeah , I know these classes is just calling com.liferay.markdown.converter.MarkdownConverter and MarkdownConverterFactoryUtil , I mean I can't find them.

**Comment by AndyWu2015** [comment]
never mind , i found them in com.liferay.knowledge.base.markdown.converter in liferay-portal repo , it was renamed.
